Mondrian-Style art in Perl!

GeekRuthie opened the pull request:
Let me know if you find anything that needs fixing here!

mdperry commented:
Oh, that is weird, @GeekRuthie, I added a comment about that Imager.pm module, but it does not seem to be here now; so I probably neglected to . . . "Save" it or something.  When I tested your code and provided an output file named "test.png" it balked and said that was not a recognized filetype name, there was a list of recognized output file types, one was .bmp, and it worked fine.  Then when I pasted the error message in my comment, I read it more carefully, and realized it was looking for  a file named PNG.pm, and couldn't find that.  Which made me suspicious.  I had to install Imager, which went smoothly.  I have not looked this up, but suspect that one may also need to install Imager::PNG (and ::JPG as well?)?

GeekRuthie commented on 2025/incoming/2025-Perl-Mondrian.pod:
```diff
@@ -0,0 +1,377 @@
+a link), 
+I'll also include a script to let you play around with the parameters of the
+generator, and output JPGs and SVGs as well as PNGs.
+
```
Yep!  I added my little command-line script to the end of the article.

GeekRuthie commented:
@mdperry, you've made some good calls here.  I've made some changes to the generator colors to be more in line with _Tableau I_, which is one of the more-famous of Mondrian's works; (He did use that pale-blue-gray in a couple of other works, but these changes make it way more consistent.  I've also changed to to be slightly less-biased toward white, and change the number of squares to paint, which will make the generator slightly-more-likely to generate colored boxes.

You don't have to reference Imager::File::PNG and Imager::File::JPEG, but they must be installed, as I found out just now when I tested it out on a fresh Perl.  So I've included them.

I've also updated the script to produce a JPG, and added a new JPG to replace the old PNG.  Update to the PR coming momentarily.

Thanks to both you, and @oalders--and this reminds me that coding and writing are best done not-at-the-same-time!  :)

mdperry reviewed (approved):
That is a very nice example that the code generated, much nicer than mine; I guess I just need to practice more(?). ;-).  Since I have zero artistic ability (maybe sometimes photography but I don't know if that counts), this module could help make anyone an artist.  I think it is good to go(!)
